service-endpoint/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# FROM python:3.11-slim-bullseye
+FROM python:3.11-alpine
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN addgroup -S app && adduser -S app -G app
+# RUN groupadd -r app && useradd --no-log-init -r -g app app
+USER app
+WORKDIR /home/app
+
+# Copy and install requirements only first to cache the dependency layer
+COPY --chown=app:app requirements.txt .
+RUN pip install --no-cache-dir --no-compile --upgrade -r requirements.txt
+
+COPY --chown=app:app . .
+
+CMD ["/home/app/.local/bin/uvicorn", "app:app", "--host", "0.0.0.0", "--port", "8000", "--proxy-headers"]
```

Comment:
We should have `--reload` here when developing @laurigates 
https://www.uvicorn.org/settings/#development
Would it be easy to add it, when we are running this locally?

Reply:
This should be possible by setting `UVICORN_RELOAD=True` in docker-compose.yml.